app/Http/Controllers/Api/Place/PlaceRatingController.php

```diff
@@ -60,36 +60,20 @@ public function setRating(SetRatingHttpRequest $httpRequest) : JsonResponse
             'id' => $setPlaceRatingResponse->getId(),
             'user_id' => $setPlaceRatingResponse->getUserId(),
             'place_id' => $setPlaceRatingResponse->getPlaceId(),
-            'rating' => $setPlaceRatingResponse->getRatingValue(),
+            'my_rating' => $setPlaceRatingResponse->getRatingValue(),
```

Comment:
Why we need my_rating here ?

Reply:
`my_rating` is clearer indication than `rating`